InternshipBE/DAL/Database/UserRoles.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DAL.Database
+{
+	public static class UserRoles
```

Comment:
move to shared library

Reply:
This class was deleted. New enum class will be created by teammates.